pkgs/by-name/re/re-isearch/package.nix

```diff
@@ -51,10 +51,10 @@ stdenv.mkDerivation {
     cp ../lib/*.so $out/lib/
```

Comment:
Replace `postInstall` with the following:

```nix
  installPhase = ''
    runHook preInstall

    mkdir -p $out/{bin,lib}

    cp ../bin/{Iindex,Isearch,Iutil,Idelete,zpresent,Iwatch,zipper} $out/bin
    cp ../lib/*.so $out/lib/

    runHook postInstall
  '';
```

Then, in the `makeFlags`, remove the `INSTALL` variable.